mono/mini/debugger-agent.c

```diff
@@ -5073,6 +5090,25 @@ ss_stop (SingleStepReq *ss_req)
 /*
  * ss_start:
  *
+ * Check if breakpoint would collide with one already in list. Print debug trace before returning if so.
+ */
+static gboolean ss_bp_is_duplicate(GSList *bps, MonoMethod *method, guint32 il_offset)
```

Comment:
Formating of the return value.

What about changing the meaning of this function to be the opposite so all checks don't need to negate it? Something like ss_bp_is_unique.